lib/index.js

```diff
@@ -3,9 +3,4 @@
 var Server = require('./server');
 
 
-// Declare internals
```

Comment:
I always have them because I am so used to expecting them there that I never need to bother check if it is declared before using.